docs/installation/linux.md

```diff
@@ -127,11 +129,59 @@ sudo apt-get remove powershell
 
 [Ubuntu 16.04]: http://releases.ubuntu.com/16.04/
 
-This works for Debian Stretch (now testing) as well.
+## Debian 8
+
+### Installation via Package Repository - Debian 8
+
+PowerShell Core, for Linux, is published to package repositories for easy installation (and updates).
```

Comment:
It might be worth adding that the Ubuntu14 deb is compatible and tested on Deb8 so that people aren't confused by ubuntu14 being in the URL